server/.env

```
@@ -0,0 +1,5 @@
+DATABASE_URL="postgresql://symie_owner:PdF9olNb1ACZ@ep-proud-band-a44m8ypd-pooler.us-east-1.aws.neon.tech/GoDo?sslmode=require&channel_binding=require"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)